image/index.css

```diff
@@ -0,0 +1,373 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.bigContainer {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  height: 100%;
+  overflow-x: hidden;
+  background-color: #f0f8ff;
+}
+
+.navbar_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  gap: 3%;
+  align-items: center;
+  height: 180px;
+  width: 100%;
+  background-color: #729762;
+  border: 1px solid #729762;
+}
+.img {
+  height: 40px;
+  width: 40px;
+  margin: 10px 8px;
+  padding: 0 20px 10px 0;
+}
+.logo_image {
+  height: 50px;
+  width: 59px;
+  border-radius: 50%;
+  align-items: center;
+  margin-top: -44%;
+}
+
+.search {
+  height: 20%;
+  width: 15%;
+  border-radius: 7%;
+  margin-left: 7%;
+  border-color: #4caf50;
+}
+
+.search::placeholder {
+  padding: 0 0 0 5%;
+}
+
+.logo_name {
+  color: #fff;
+  font-size: larger;
+  font-weight: 900;
+}
+.anchor {
+  text-decoration: none;
+  color: #fff;
+}
+.anchor:hover {
+  color: #0c08f1;
+}
+.btn {
+  height: 24%;
+  width: 7%;
+  border-radius: 20%;
+  border-color: #4caf50;
+}
+.btn:hover {
+  background-color: #0c08f1;
+}
+
+.contentBox {
+  display: flex;
+  flex-direction: column;
+  background-color: #f0f8ff;
+}
+
+.item {
+  display: flex;
+  justify-content: space-around;
+  border: 1px solid #4caf50;
+  margin: 1% 1% 1% 1%;
+  border-radius: 1%;
+  height: 280px;
+  padding: 2%;
+  margin: 7px;
+}
+.item:hover {
+  background-color: #4caf50;
+}
+.ingredient_ul {
+  list-style-type: none;
+}
+.rb_img {
+  height: 56%;
+  width: 90%;
+  border-radius: 6%;
+  transition: all 0.6s linear 0s;
+}
+
+.rightBox {
+  position: relative;
+  display: inline-block;
+}
+
+.rightBox:hover .rb_img {
+  transform: scale(1.2);
+  cursor: pointer;
+}
+
+.recipe_li {
```

Comment:
Give a proper name for according to the usage of the tag or purpose
It should be like `recipe_li` -> `recipe_list`

**Also check other places** 
should not use btn, img, li, ul, rg, og, etc these short hands